src/coreComponents/physicsSolvers/fluidFlow/FlowSolverBase.cpp

```diff
@@ -736,7 +736,7 @@ void FlowSolverBase::prepareStencilWeights( DomainPartition & domain ) const
     FiniteVolumeManager const & fvManager = numericalMethodManager.getFiniteVolumeManager();
     FluxApproximationBase const & fluxApprox = fvManager.getFluxApproximation( getDiscretizationName() );
     ElementRegionManager::ElementViewAccessor< arrayView1d< real64 const > > hydraulicAperture =
-      mesh.getElemManager().constructViewAccessor< array1d< real64 >, arrayView1d< real64 const > >( fields::flow::hydraulicAperture::key() );
+      mesh.getElemManager().constructViewAccessor< array1d< real64 >, arrayView1d< real64 const > >( fields::flow::aperture0::key() );
```

Comment:
aperture0 is not the aperture at the previous time step. I don't think this makes sense. I guess that what you would like is to use `aperture_n`.

Reply:
aperture0 is aperture_n, no?
https://github.com/GEOS-DEV/GEOS/blob/decc28110acf58053a0d6911f5ea7a4d9d7f76d1/src/coreComponents/physicsSolvers/fluidFlow/FlowSolverBaseFields.hpp#L147

Reply:
`aperture0` is the aperture at the previous time step. It is updated here in `SinglePhaseBase::implicitStepSetup` function.

https://github.com/GEOS-DEV/GEOS/blob/decc28110acf58053a0d6911f5ea7a4d9d7f76d1/src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp#L664C1-L666C57

Reply:
ah yeah, my bad. We should change the name tbh so that it is the same as all other variables. I would do it separately, anyways.